hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/upgrade/DataNodeUpgradeFinalizer.java

```diff
@@ -41,63 +42,58 @@
     BasicUpgradeFinalizer<DatanodeStateMachine, HDDSLayoutVersionManager> {
   private static final Logger LOG =
```

Comment:
I believe we would be fine with the logger instance defined in the UpgradeFinalizer interface, as any of the actual finalizers will log to their respective component's log file.

Reply:
Fixed